iotdb-client/client-py/iotdb/utils/Field.py

```diff
@@ -61,71 +58,73 @@ def get_data_type(self):
     def is_null(self):
         return self.__data_type is None
 
-    def set_bool_value(self, value):
-        self.__bool_value = value
+    def set_bool_value(self, value: bool):
+        self.value = value
 
     def get_bool_value(self):
         if self.__data_type is None:
             raise Exception("Null Field Exception!")
-        return self.__bool_value
+        if self.__data_type != TSDataType.BOOLEAN:
+            return None
+        return self.value
 
-    def set_int_value(self, value):
-        self.__int_value = value
+    def set_int_value(self, value: int):
+        self.value = value
 
     def get_int_value(self):
         if self.__data_type is None:
             raise Exception("Null Field Exception!")
-        return self.__int_value
+        if self.__data_type != TSDataType.INT32:
+            return None
+        return np.int32(self.value)
 
-    def set_long_value(self, value):
-        self.__long_value = value
+    def set_long_value(self, value: int):
+        self.value = value
 
     def get_long_value(self):
         if self.__data_type is None:
             raise Exception("Null Field Exception!")
-        return self.__long_value
+        if self.__data_type != TSDataType.INT64:
+            return None
+        return np.int64(self.value)
 
-    def set_float_value(self, value):
-        self.__float_value = value
+    def set_float_value(self, value: float):
+        self.value = value
 
     def get_float_value(self):
         if self.__data_type is None:
             raise Exception("Null Field Exception!")
-        return self.__float_value
+        if self.__data_type != TSDataType.FLOAT:
+            return None
+        return np.float32(self.value)
```

Comment:
`np.float32` would return `np.nan`. Is this an expected result?

Reply:
No. If self.value == None, I think the result should be None. Added check for the self.value.